src/web/components/SlotBodyEnd/BrazeEpic.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react';
+import * as emotion from 'emotion';
+import * as emotionCore from '@emotion/core';
+import * as emotionTheming from 'emotion-theming';
+import {
+	parseBrazeEpicParams,
+	EpicDataFromBraze,
+	Variant,
+} from '@root/src/web/lib/braze/parseBrazeEpicParams';
+import { BrazeMessagesInterface } from '@root/src/web/lib/braze/BrazeMessages';
+import { CanShowResult } from '@root/src/web/lib/messagePicker';
+import { useOnce } from '@root/src/web/lib/useOnce';
+import { joinUrl } from '@root/src/lib/joinUrl';
+
+const { css } = emotion;
+
+const wrapperMargins = css`
+	margin: 18px 0;
+`;
+
+const EPIC_COMPONENT_PATH = '/epic.js';
```

Comment:
if it is only used once, just set the string. Add a comment to add more detail if needed